### ISSUE TITLE
Content missing after heading - SiteImprove - Stat

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa801#8508]
# Problem 

Siteimprove is flagging an issue with the stat component, **Content missing after heading**.

## **Content missing after heading**

The stat component uses a heading tag but the heading information is not very useful as a heading.   On pages like https://uiowa.edu/about-iowa with a headline before the stats, the **Content missing after heading** is reported. 

Example document outline with the stat component:

<img width="336" alt="Image" src="https://github.com/user-attachments/assets/079cc885-0f31-4b2f-91d6-77f0642a8397" />

### Possible solution

- Remove heading markup options from stat component?

**Comment by cory-skeers** [comment]
Both of these issues came up in recent SI investigations on uiowa.edu: https://my2.siteimprove.com/Accessibility/995976/NextGen/Issue/1?ruleId=72&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2 and https://my2.siteimprove.com/Accessibility/995976/NextGen/Issue/1?ruleId=78&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

**Comment by joewhitsitt** [comment]
I am going to split the text all caps off into a separate issue
